packages/eslint-plugin/src/configs/core.js

```diff
@@ -0,0 +1,357 @@
+// @ts-check
+const configHelpers = require('../utils/configHelpers');
```

Comment:
seems that this should be the `base` and then have `core` and `core-legacy`. At least that makes more sense to me.

Not blocking tho

Reply:
hmm you mean changing the names only or the inheritance chain as well ?